crates/rpc/src/websocket.rs

```diff
@@ -0,0 +1,20 @@
+use crate::module::Module;
+
+use starknet_gateway_types::websocket::WebsocketSenders;
+
+pub mod subscription;
+
+pub fn register_subscriptions(
+    module: Module,
+    ws_broadcast_txs: WebsocketSenders,
+) -> anyhow::Result<Module> {
+    let module = module.register_subscription(
+        "starknet_subscribe_newHeads",
+        "s_newHeads",
+        "starknet_unsubscribe_newHeads",
+        subscription::subscribe_new_heads::subscribe_new_heads,
```

Comment:
We can't use `starknet_xxx` as that implies its some official part of the starknet RPC specification -- which this isn't yet. As such it should be a pathfinder extension i.e. `pathfinder_xxx`.

Reply:
Aye sir, sure can.